core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -291,6 +289,8 @@ angular.module('oppia').directive('storyNodeEditor', [
               controller: [
                 '$scope', '$uibModalInstance',
                 function($scope, $uibModalInstance) {
+                  $scope.MAX_CHARS_IN_CHAPTER_TITLE =
```

Comment:
Please wrap inside `(..)` and break after `(` here.

Reply:
Done